moto/core/exceptions.py

```diff
@@ -3,8 +3,6 @@
 from typing import Any, List, Tuple, Optional
 import json
 
-# TODO: add "<Type>Sender</Type>" to error responses below?
-
 
 SINGLE_ERROR_RESPONSE = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This doesn't seem to contain the `Type` field

<https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html>